fetch-icon.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+CURRENT_DIR=$(pwd)
```

Comment:
@ankit-kumar-dwivedi I suggest to replace that with this:
```CURRENT_DIR=$(cd `dirname "$0"`; pwd)```

In this case the script will work even if we launch it from outside of repository directory. ^__^

Reply:
Done !